cordapp/contracts/src/main/java/com/newamerica/contracts/FundContract.java

```diff
@@ -50,8 +50,8 @@ public void verify(LedgerTransaction tx) {
                 require.using("The amount must be greater than zero.", outputState.amount.compareTo(BigDecimal.ZERO) > 0);
                 require.using("The balance must be greater than zero.", outputState.balance.compareTo(BigDecimal.ZERO) > 0);
                 require.using("The balance and amount fields must be equal during an issuance.", outputState.getAmount().compareTo(outputState.getBalance()) == 0);
-                require.using("The maxWithdrawalAmount must be greater than or equal to zero.", outputState.maxWithdrawalAmount.compareTo(BigDecimal.ZERO) > 0);
```

Comment:
=  to >=

Reply:
This is done already. Check the added code on line 53